solr/core/src/test-files/runtimecode/MyPlugin.java

```diff
@@ -1,43 +0,0 @@
-/*
```

Comment:
This file is accidentally deleted

Reply:
Fixed. This file was causing problems in Eclipse. We should deal with it separately.